### DISCUSSION TITLE
post/gitpod-is-in-big-trouble

### DISCUSSION THREAD

**Discussion by giscus** [zihan-ch/blog#33] [General]
# post/gitpod-is-in-big-trouble

About the new free plan of Github Codespaces and how Gitpod got into big trouble.

http://localhost:3000/post/gitpod-is-in-big-trouble

**Comment by arstnei0** [comment]
See [this article on HN](https://news.ycombinator.com/item?id=33569604) to find the comments(Even though some people like to say my article is trash, I still prefer to share the link)
